packages/field-views-loader/index.js

```diff
@@ -100,11 +99,11 @@ module.exports = function() {
 
   let loaders = `{\n${[...allPaths]
     .map(path => {
-      return `'${path}': () => import('${path}').then(interopDefault)`;
+      return `${JSON.stringify(path)}: () => import(${JSON.stringify(path)}).then(interopDefault)`;
     })
     .join(',\n')}\n}`;
 
-  const source = endent`
+  const source = `
```

Comment:
endent was breaking things and removing it fixed it, I know having it is nice but debugging things on Windows is annoyyyyyyyyying and I didn't want to do any more.